catalog/core/catalog-core-api/src/main/java/ddf/catalog/data/Result.java

```diff
@@ -23,33 +23,33 @@
 public interface Result {
 
   /** Constant used to specify a sort policy based on relevance. */
-  public static final String RELEVANCE = "RELEVANCE";
```

Comment:
Don't think we can change these api constants or we will break backwards compatibility. Might be a better change for DDF 3.0

Reply:
You're right. This is whitelisted: http://www.codice.org/ddf/documentation.html#_catalog_whitelist. I'll update this PR to just suppress that finding.